src/pages/dashboard/[subdomain]/editor.tsx

```diff
@@ -174,12 +183,18 @@ export default function SubdomainEditor() {
     <DashboardLayout title="Editor">
       <DashboardMain fullWidth>
         <header className="flex justify-between absolute top-0 left-0 right-0 z-10 px-5 h-14 border-b items-center text-sm">
-          <EditorToolbar view={view} toolbars={toolbars} preview={preview} modeToolbars={modeToolbars}></EditorToolbar>
+          <EditorToolbar
+            view={view}
+            toolbars={toolbars}
+            preview={previewRef}
+            previewVisible={previewVisible}
```

Comment:
how's `previewRef` different from just passing `previewVisible` and `setPreviewVisible` to the component? 🤔